RestaurantApp/Pages/MainPage.xaml.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using RestaurantApp.Models;
+using Xamarin.Forms;
+
+namespace RestaurantApp.Pages
+{
+    public partial class MainPage : MasterDetailPage
+    {
+        public MainPage()
+        {
+            InitializeComponent();
+
+            MasterPage.listView.ItemSelected += OnItemSelected;
```

Comment:
you shouldn't subscribe to events in code behind, you can do it right from xaml like this:

<ListView ItemSelected="OnItemTapped"></ListView>

and in xaml.cs

private void OnItemTapped(object sender, ItemTappedEventArgs e)
        {
            handle here
        }